src/test/java/chess/BishopTest.java

```diff
@@ -23,9 +25,81 @@ public void testBlackBishopToString() {
     @Test
     public void testWhiteBishopToString() {
         Position pos = new Position(0, 0);
-        Bishop piece = new Bishop(Color.WHITE, pos);
+        ChessBoard chessBoard = new ChessBoard();
+        Bishop piece = new Bishop(Color.WHITE, pos, chessBoard);
         String expected = "BW";
         String actual = piece.toString();
         assertEquals(expected, actual);
     }
+
+    @Test
+    public void testMoveLeftDownBishop() {
+        ChessBoard chessBoard1 = new ChessBoard();
+        chessBoard1.crateAnEmptyChessBoard();
+        ChessBoard chessBoard = new ChessBoard(new Bishop(Color.WHITE,new Position(4,4),chessBoard1));
+        Bishop piece = (Bishop) chessBoard.getBoard()[4][4];
+        boolean po= piece.move(new Position(0,0));
+        assertTrue(po);
```

Comment:
please add spaces after each coma

Reply:
i made the changes could you review it please?